### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by de1mat** [viezel/NappSlideMenu#35]
I am using this great module in my app without problems - I thought.

Now that my app has been released I am having users report that they cannot use the app due to VoiceOver not working for them. These users have visual impairments.

What I discovered from a quick play is that when my app is opened the VoiceOver works fine - until I open a fresh Window that is using the NappSlideMenu. At that point the accessibility features of my app cease working and VoiceOver does nothing. In fact some buttons while in VoiceOver mode become completely useless. With VoiceOver "off" all is well.

Any other developers experienced this? I am wondering if accessibility was tested as part of this module? I know I didn't look at this until now :/

I will play around some more and see what I find, but I thought I would post this and see what others have to say...

**Comment by dieg013** [comment]
Hello, I have the same problem you have to find a solution?

Thank

**Comment by de1mat** [comment]
I did not find the solution sorry.
